src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7115,6 +7157,43 @@ private BoundExpression BindIndexedPropertyAccess(SyntaxNode syntax, BoundExpres
 
                 if (!analyzedArguments.HasErrors)
                 {
+                    // PROTOTYPE: REMOVE BEFORE SHIPPING DEV16
+                    // Pretend like there are indexers that support range and index
+                    if (receiverOpt?.Type.SpecialType == SpecialType.System_String &&
+                        analyzedArguments.Arguments.Count == 1)
+                    {
+                        HashSet<DiagnosticInfo> diags = null;
+                        var argType = analyzedArguments.Arguments[0].Type;
+                        TypeSymbol resultType = null;
+                        if (argType == GetWellKnownType(WellKnownType.System_Index, ref diags))
```

Comment:
>GetWellKnownType(WellKnownType.System_Index, ref diags) [](start = 39, length = 55)

Since we're not reporting use site diagnostics, consider using `Compilation.GetWellKnownType(type)` for these two calls.